calcom/cal.com
# fix: phone selector dropdown hidden behind logo on booking page

## What does this PR do?

On the booking page, when there is a phone field, the logo of cal.com appears in front of the phone field dropdown menu. 

<img width="633" alt="image" src="https://github.com/calcom/cal.com/assets/138862352/83667b64-d90b-4d7f-9fea-75cef28f0902">

### Fixes:

This PR fixes z-index of cal.com logo in the booking page, so that it doesn’t appear in front of dropdown menu of a phone field.

**Before:**

https://www.loom.com/share/902a478bae934c9b97057fe022175375?sid=db1762bd-9fa8-4b7b-83ea-c60f11f15829

**After:**

https://www.loom.com/share/16d3e089c04f47e7af62b5ba1f367231?sid=50f81960-2016-4796-950f-47f8527cd599

## Type of change


## How should this be tested?

1. Create an Event Type with a "Phone number" booking question.
2. Visit booking page
3. Click on the phone selector and verify that it's in front of the logo

## Mandatory Tasks


## Checklist



- [x] I have checked if my PR needs changes to the documentation
- [x] I have checked if new and existing unit tests pass locally with my changes
